crates/re_space_view_spatial/src/eye.rs

```diff
@@ -260,7 +352,28 @@ impl OrbitEye {
 
     /// Returns `true` if interaction occurred.
     /// I.e. the camera changed via user input.
-    pub fn update(&mut self, response: &egui::Response, drag_threshold: f32) -> bool {
+    pub fn update(
+        &mut self,
+        response: &egui::Response,
+        drag_threshold: f32,
+        bounding_boxes: &SceneBoundingBoxes,
+    ) -> bool {
+        let mut speed = match self.mode {
+            EyeMode::FirstPerson => 0.1 * bounding_boxes.accumulated.size().length(), // TODO(emilk): user controlled speed
```

Comment:
Wondering if the `current` bounding box (or a smoothed version :thinking:) wouldn't be a better measure for this. `accumulated` often (albeit not always :/) contains outliers and when navigating around one often does so when the scene is at rest